plugins/chunter-resources/src/components/Message.svelte

```diff
@@ -59,6 +60,23 @@
         action: chunter.actionImpl.SubscribeMessage
       } as Action)
 
+  async function deleteMessage () {
+    if (space) {
+      await client.removeDoc(chunter.class.Message, space, message._id)
+      const loc = getCurrentLocation()
+
+      if (loc.path.length > 3) {
+        loc.path.length = 3
```

Comment:
Why we need current url?

Reply:
You should check it before close thread `if (loc.path[3] === message._id)`

Reply:
Done